### PR TITLE
SB/27-Change-User-Profile-Type

### PR DESCRIPTION
**Pull request by shanedbutler**
# Description

This PR adds the ability for an admin to change a user profile type to and from admin or author.

# Testing Instructions

1. Launch application from SB/27-Change-User-Profile-Type branch
2. In the app, Login as admin@example.com
3. Go to "Users" on the nav bar
5. You should be greeted with a list of users from the database's UserProfile table
6. Follow a user's pencil icon button to navigate to a change type page
7. On this page you should be able to change user type to and from admin or author
8. Upon submitting a change you will be redirected back to the list of users where you can verify the change

If you don't have data for the UserProfile table, you may utilize this SQL query to seed 3 new users to table.
```SQL
INSERT INTO [UserProfile] 
([FirstName], [LastName], [DisplayName], [Email], [CreateDateTime], [ImageLocation], [UserTypeId])
VALUES 
('Jess', 'Ximenez', 'ximjess', 'jess8000@yahoo.com', SYSDATETIME(), 'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png', 2),
('Rick', 'Rust', 'rustyrick', 'rstnvrslps@hotmail.com', SYSDATETIME(), 'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png', 2),
('Bianca', 'Bolander', 'bboland', 'berezepz@gmail.com', SYSDATETIME(), 'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909_960_720.png', 2);
```

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by sarahebrooks12** [comment]
Testing instructions look good --- get 2 testing approvals before merging 👼

**Review by TabathaSlater — APPROVED** [review]
Looks good and works well on my end

**Pull request merged**
